code/modules/mob/living/carbon/xenomorph/evo_datum.dm

```diff
@@ -0,0 +1,83 @@
+/// Global variable for accessing evolution panel. Does not need to be unique to each xeno as ui_data pulls relevant information based on context.
+GLOBAL_DATUM_INIT(evo_panel, /datum/evolution_panel, new)
```

Comment:
Just move this to code/_globalvars/tgui.dm pls

Reply:
Done